pom.xml

```diff
@@ -285,6 +285,11 @@
         <artifactId>kubernetes-model-flowcontrol</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>io.fabric8</groupId>
+        <artifactId>kubernetes-model-gatewayapi</artifactId>
```

Comment:
Shouldn't we add this module to:
- uberjar pom.xml
- uberjar with reduced dependencies pom.xml
- Karaf features (and maybe a test too)

?